main.go

```
@@ -1,17 +1,33 @@
-    r.Use(middleware.Logger)
-    r.Get("/", func(w http.ResponseWriter, r *http.Request) {
-        w.Write([]byte("Hello World!"))
-    })
-    http.ListenAndServe(":3000", r)
+
+	db, err := gorm.Open(sqlite.Open("db/development.db"), &gorm.Config{})
+	if err != nil {
+		panic("failed to connect database")
+	}
+
+	db.AutoMigrate(&models.Feature{})
```

golangci-lint (1.64.8)

Error return value of `db.AutoMigrate` is not checked

(errcheck)